.github/workflows/send-screenshots-to-pixeleagle.yml

```diff
@@ -16,18 +16,33 @@ on:
         required: true
         type: string
 
+env:
+  # Unfortunately, we can't check secrets in `if:` conditionals. However, Github's own documentation
+  # suggests a workaround: Putting the secret in an environment variable, and checking that instead.
+  PIXELEAGLE_TOKEN_EXISTS: ${{ secrets.PIXELEAGLE_TOKEN != '' }}
+
 jobs:
   send-to-pixel-eagle:
     name: Send screenshots to Pixel Eagle
     runs-on: ubuntu-24.04
+    # Pixel Eagle is irrelevant for most forks, even of those that allow workflows to run. Thus, we
+    # disable this job for any forks. Any forks where Pixel Eagle is relevant can comment out the
+    # `if:` conditional below.
+    if: ${{ github.repository == 'bevyengine/bevy' }}
     steps:
+      - name: Notify user on non-existent token
+        if: ${{ ! fromJSON(env.PIXELEAGLE_TOKEN_EXISTS) }}
```

Comment:
```suggestion
```

Reply:
If I were to make this change, I would have to remove this entire step. There is no point to running this step unless `secrets.PIXELEAGLE_TOKEN == ''`.